packages/ui-components/src/components/tag/tag.tsx

```diff
@@ -0,0 +1,22 @@
+import { Component, Host, Prop, h } from '@stencil/core';
+
+@Component({
+	tag: 'kv-tag',
+	styleUrl: 'tag.scss',
+	shadow: true
+})
+export class KvTag {
+	/** Tag text */
```

Comment:
I believe we're missing `(required)` for proper documentation generation

Reply:
Yeah! nice catch! However, here I will add the (optional) instead. Since we have the slot there, this text might not even be necessary.